### PR TITLE
TINY-10799: Color picker input field error tooltip.

### PR DESCRIPTION
**Pull request by HAFRMO**
Related Ticket: https://ephocks.atlassian.net/browse/TINY-10799

Description of Changes:
Add an error tooltip and error icon when the text input is not valid.

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

**Review by TheSpyder — CHANGES_REQUESTED** [review]
I posted the details in JIRA for wider comment, but we can't go inventing new error messaging. The editor-wide system for doing this is the `alertbanner` dialog component.

**Review by spocke — APPROVED** [review]

File: modules/oxide/src/less/theme/components/color-picker/color-picker.less [hunk 1]
```diff
@@ -99,12 +99,43 @@
...
+    top: 0;
+
+    .tox-control-wrap__status-icon-invalid {
+      margin: 0;
+    }
+  }
+
+  .tox-rgb-warning-note {
+    background-color: #fcc;
+    border: 1px solid #f00;
```
[the remark this answers is not in the document]
Comment (HAFRMO) [reply modules/oxide/src/less/theme/components/color-picker/color-picker.less:131]: Maybe? We didn't really establish what colors these should be. I can move them to some variables though. Should I keep it in color-picker.less or somewhere else?

File: modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts [hunk 2]
```diff
@@ -20,6 +21,7 @@ const english: Record<string, string> = {
   'colorcustom.rgb.hex.label': '#',
   'colorcustom.rgb.hex.description': 'Hex color code',
   'colorcustom.rgb.range': 'Range 0 to 255',
+  'colorcustom.rgb.invalid': 'Numbers only, 0 to 255',
```
[the remark this answers is not in the document]
Comment (HAFRMO) [reply modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts:24]: Do we have any notion-page or wider issue for this, or should I just make one specifically for this?

[the remark this answers is not in the document]
Comment (HAFRMO) [reply modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts:24]: Added a ticket.

**Review by ztomaszyk — APPROVED** [review]
It would be good to write some tests to verify the icon is showing on incorrect value

**Pull request merged**